addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -21,7 +21,7 @@
                 <t t-if="chatWindow.thread and chatWindow.thread.mass_mailing">
                     <i class="fa fa-envelope-o" title="Messages are sent by email" role="img"/>
                 </t>
-                <t t-if="chatWindow.thread and chatWindow.thread.localMessageUnreadCounter > 0">
+                <t t-if="chatWindow.thread and chatWindow.thread.localMessageUnreadCounter > 0 and !chatWindow.thread.isDisabled">
```

Comment:
I don't understand this condition, displaying `localMessageUnreadCounter` is not related to thread being disabled.